servicetalk-buffer-netty/src/main/java/io/servicetalk/buffer/netty/ServiceTalkBufferAllocator.java

```diff
@@ -159,6 +159,17 @@ public Buffer wrap(byte[] bytes) {
                 bytes.length));
     }
 
+    @Override
+    public Buffer wrap(final byte[] bytes, final int offset, final int len) {
+        if (len == 0) {
+            return EMPTY_BUFFER;
+        }
+        final Buffer b = wrap(bytes);
+        b.readerIndex(offset);
+        b.writerIndex(offset + len);
```

Comment:
In my minds eye this might be an unexpected behavior change: the resulting buffer will have different reader and writer indexes than expected as if they had sliced the `byte[]` themselves beforehand. This could be a trip hazard as users often will reset the reader index to 0 to reuse the buffer efficiently.